ng/src/web/pages/filters/dialog.js

```diff
@@ -61,8 +60,8 @@ const FilterDialog = ({
       onClose={onClose}
       onSave={onSave}
       defaultValues={{
-        ...DEFAULTS,
         ...filter,
```

Comment:
Same as for the ScheduleDialog. I suppose only id is used from the filter here. Then passing the id explicitly would be better.

Reply:
In this case, id and comment were used from the filter. It still seems more efficient to just pass those two props instead of the whole filter, so I changed it.